docs/json_rpc_api.mediawiki

```diff
@@ -762,6 +1289,29 @@ the method name for further details such as parameter and return information.
 
 ----
 
+====getcurrentnet====
+{|
+!Method
+|getcurrentnet
+|-
+!Parameters
+|None
+|-
+!Description
+|Get Decred network dcrd is running on.
+|-
+!Returns
+|<code>numeric</code>
+|-
+!Example Return
+|
+: <code>(mainnet)</code> <code>3652501241</code>
+: <code>(testnet)</code> <code>118034699</code>
+: <code>(testnet2)</code> <code>1223139429</code> 
```

Comment:
Since this is being update, should replace the `testnet` and `testnet2` examples with `testnet3` and the appropriate value here instead.